Server/validation/joi.js

```diff
@@ -35,7 +35,7 @@ const registerValidation = joi.object({
     .string()
     .required()
     .pattern(/^[A-Za-z]+$/),
-  email: joi.string().email().required(),
+  email: joi.string().email().lowercase().required(),
```

Comment:
This validation isn't working, not sure why.

I can still create a user with uppercase chars in the email.

We'll have to investigate this further